Dockerfile

```diff
@@ -20,6 +20,19 @@ RUN sed -i 's/^\($ModLoad imklog\)/#\1/' /etc/rsyslog.conf
 # Fix potential UTF-8 errors with ansible-test.
 RUN locale-gen en_US.UTF-8
 
+# Cleanup unwanted systemd files
+# See https://hub.docker.com/_/centos/ and https://github.com/ansible/molecule/issues/1104
+RUN find /lib/systemd/system/sysinit.target.wants/* ! -name systemd-tmpfiles-setup.service -delete; \
+rm -f /lib/systemd/system/multi-user.target.wants/*;\
+rm -f /etc/systemd/system/*.wants/*;\
+rm -f /lib/systemd/system/local-fs.target.wants/*; \
+rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+rm -f /lib/systemd/system/basic.target.wants/*;\
+rm -f /lib/systemd/system/anaconda.target.wants/*; \
+rm -f /lib/systemd/system/systemd*udev*; \
+rm -f /lib/systemd/system/getty.target
```

Comment:
Why the multiple `rm`s ?

Reply:
This was just a rebase of #12 where they explained those files need to be removed to fix problems with systemd in a container running in a systemd system. The biggest problem is the container causes the container to log the user of the host out when it stops. Why multiple `rm`s instead of a single `rm`. No real reason I was just rebasing the pull because @geerlingguy asked the original guy to rebase but the original guy wasn't responding. Same goes for the whitespace inconsistency.